frontend/workflows/projectSelector/src/project-group.tsx

```diff
@@ -153,51 +156,54 @@ const ProjectGroup: React.FC<ProjectGroupProps> = ({ title, group, displayToggle
           {numProjects === 0 && (
             <StyledNoProjectsText>No projects in this group yet.</StyledNoProjectsText>
           )}
-          {Object.keys(state[group]).map(key => (
-            <StyledMenuItem key={key}>
-              <Checkbox
-                name={key}
-                size="small"
-                disabled={state.loading}
-                onChange={() =>
-                  dispatch({
-                    type: "TOGGLE_PROJECTS",
-                    payload: { group, projects: [key] },
-                  })
-                }
-                checked={!!state[group][key].checked}
-              />
-              <StyledMenuItemName>{key}</StyledMenuItemName>
-              <StyledHoverOptions hidden>
-                <StyledOnlyButton
-                  onClick={() =>
-                    !state.loading &&
-                    dispatch({
-                      type: "ONLY_PROJECTS",
-                      payload: { group, projects: [key] },
-                    })
-                  }
-                >
-                  Only
-                </StyledOnlyButton>
-                <StyledClearIcon>
-                  {state[group][key].custom && (
-                    <IconButton
+          {Object.keys(state[group]).map(
+            key =>
+              !deriveHiddenStatus(state, group, key) && (
+                <StyledMenuItem key={key}>
```

Comment:
no changes from this point on; the rest are just indentation updates b/c of the addition of line `161`